modules/service/src/main/scala/lucuma/odb/api/service/Routes.scala

```diff
@@ -4,10 +4,11 @@
 package lucuma.odb.api.service
 
 import lucuma.odb.api.service.ErrorFormatter.syntax._
+import _root_.fs2.Stream
```

Comment:
Why do you need the `_root_`. Do we have an internal `fs2` package?

Reply:
You're right it isn't necessary here.  That was a habit from when I had placed the FS2/Sangria streaming integration in an `fs2` package there.